frontend/src/app/choreography/groups/four-group/four-group.component.html

```diff
@@ -0,0 +1,28 @@
+<mat-form-field>
```

Comment:
**suggestion:** can this be extracted into a component like `<app-group-person-selector key="eturi" (selectPerson)="group.eturi = $event"></app-group-person-selector>`